addons/l10n_pt/models/chart_template.py

```diff
@@ -0,0 +1,40 @@
+from odoo import models, _, api
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    def _get_account_vals(self, company, account_template, code_acc, tax_template_ref):
+        vals = super()._get_account_vals(company, account_template, code_acc, tax_template_ref)
+        if company.account_fiscal_country_id.code != 'PT':
+            return vals
+        vals['l10n_pt_taxonomy_code'] = account_template.l10n_pt_taxonomy_code
+        return vals
```

Comment:
`l10n_pt_taxonomy_code` is supposed to be set only on pt, right? So
```suggestion
    def _get_account_vals(self, company, account_template, code_acc, tax_template_ref):
        vals = super()._get_account_vals(company, account_template, code_acc, tax_template_ref)
        vals['l10n_pt_taxonomy_code'] = account_template.l10n_pt_taxonomy_code
        return vals
```